MediaPicker/src/main/java/org/wordpress/android/mediapicker/loader/DeviceListBuilder.kt

```diff
@@ -44,7 +42,7 @@ class DeviceListBuilder(
         if (!loadMore) {
             cache.clear()
         }
-        val lowerCaseFilter = filter?.toLowerCase(localeManagerWrapper.getLocale())
+        val lowerCaseFilter = filter?.lowercase()
```

Comment:
This is how the implementation looks like:

![image](https://user-images.githubusercontent.com/5845095/126483821-070409ef-f512-4bb4-ae26-f4538e48efd8.png)